src/DatePicker/CalendarToolbar.js

```diff
@@ -72,13 +76,17 @@ class CalendarToolbar extends Component {
       year: 'numeric',
     }).format(displayDate);
 
+    const nextButtonIcon = this.context.muiTheme.isRtl ? <NavigationChevronLeft /> : <NavigationChevronRight />;
+    const prevButtonIcon = this.context.muiTheme.isRtl ? <NavigationChevronRight /> : <NavigationChevronLeft />;
+
+
     return (
       <div style={styles.root}>
```

Comment:
If you add "direction: ltr;" for the toolbar while the direction of the theme/page is rtl, you mess up the placement of the icons as well as the placement of the text in between them.

The "next" icon should always point "forward", or, in the "to" direction of the text.  "Next" should point left for RTL and point right for LTR. It cannot be invariant from the direction logic.